R/as_model_out_df.R

```diff
@@ -0,0 +1,211 @@
+#' Convert model output to a `as_model_out_tbl` class object.
+#'
+#' @param tbl a `data.frame` or `tibble` of model output data returned from a query
+#' to a `<hub_connection>` object.
+#' @param model_id_col character string. If an `model_id` column does not
+#' already exist in `tbl`, the `tbl` column name containing `model_id` data.
+#' Alternatively, if both a `team_abbr` and a `model_abbr` column exist, these will
+#' be merged automatically to create a single `model_id` column.
+#' @param output_type_col character string. If an `output_type` column does not
+#' already exist in `tbl`, the `tbl` column name containing `output_type` data.
+#' @param output_type_id_col character string. If an `output_type_id` column does not
+#' already exist in `tbl`, the `tbl` column name containing `output_type_id` data.
+#' @param value_col character string. If an `value` column does not
```

Comment:
```suggestion
#' @param value_col character string. If a `value` column does not
```

Reply:
Fixed in 45d3c0da56e26292c93ff8198fa310184f2239d5 👍